_pages/install/integrations/configurable-products-integration.md

```diff
@@ -80,6 +80,12 @@ After running this command, you are asked which product configurator features yo
 
 The saved cart feature is generally supported with the Configurable Products integration. A saved cart can contain a configurable product and can be activated. After the cart is activated, the configuration can be accessed and edited. Note, however, that as long as the saved cart is not activated, the configuration of the configurable product cannot be displayed.
 
+## Cart Validation
+
+Cart validation needs to be enabled explicitly, see [{% assign linkedpage = site.pages | where: "name", "cart-validation.md" %}{{ linkedpage[0].title }}]({{ site.baseurl }}{% link _pages/dev/features/cart-validation.md %}). That is possible only if the commerce release is 2011 or higher. If cart validation is enabled, it’s not possible to checkout carts with configurations that are conflicting or that contain mandatory attributes without values selected. 
+In case your commerce release is 2005 or lower, consider to disable the checkout button, see `cart-totals.component`, depending on possible issues of the cart bound configurations. Use `ConfiguratorCartServiceactiveCartHasIssues` for that.
```

Comment:
@gladius-mtl I just checked that we should not have a link to the older versions here, because the cart has been refactored extensively (the checkout button is in its own component now).
What about phrasing it like this:
In case your commerce release is 2005 or lower, consider to disable the checkout button in cart-proceed-to-checkout.component.html in case ConfiguratorCartService#activeCartHasIssues emits true